### PR TITLE
feat: add post, all post page, get/post fetch post's

### PR DESCRIPTION
**Pull request by RayzigenSchwarz**
add all post page
Simulator Screen Shot - iPhone 13 - 2022-06-10 at 16 06 39
add post component
add post request 
add get request 
add store for post
add edit post pag
Simulator Screen Shot - iPhone 13 - 2022-06-10 at 16 07 29
e

### CODE REVIEW

File: src/assets/icon/AddPostIcon.tsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import Svg, {Path} from 'react-native-svg';
 
 const AddPostIcon = () => {
```
Comment (Ruslan1516) [inline src/assets/icon/AddPostIcon.tsx:4]: add props {width, height} and set default value

File: src/screen/CreatePost/CreatePost.tsx [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+  const dispatch = useDispatch();
+
+  setStorage(
+    'token',
+    'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjYyOGM2YjljZmYwOGJmMDAwNGFmOWZmNCIsInVzZXIiOiJzbWFsbGRpY2sxMjMiLCJpYXQiOjE2NTMzNjk3NTZ9.6mAu9BZTY5aY1g8OMSRxO47kEj5eVLUndTwzZ0UVDGQ',
+  );
+
+  const onSubmit = (values: PostsTypes) => {
+    dispatch(fetchPostsRequest(values));
+    console.log(values);
```
Comment (Ruslan1516) [inline src/screen/CreatePost/CreatePost.tsx:25]: remove bleat

File: src/screen/CreatePost/CreatePost.tsx [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+                  value={valuesDescription}
+                />
+
+                {descriptionError && (
+                  <Text style={style.alertErrorShow}>{descriptionError}</Text>
+                )}
+
+                <TouchableOpacity
+                  style={style.buttonStyle}
+                  // underlayColor={COLORS.RED}
```
Comment (Ruslan1516) [inline src/screen/CreatePost/CreatePost.tsx:107]: please remove commented code, it messes up the project

File: src/screen/AllPosts/AllPost.tsx [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+                userAvatar={
+                  'https://ps.w.org/user-avatar-reloaded/assets/icon-256x256.png?rev=2540745'
+                }
+              />
+            </View>
+          );
+        }}
+        keyExtractor={(post: postType) => post._id}
+      />
+      {/*)}*/}
```
Comment (Ruslan1516) [inline src/screen/AllPosts/AllPost.tsx:50]: WTF?

File: src/screen/AllPosts/AllPost.tsx [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+  useEffect(() => {
+    dispatch(fetchGetUserinfoRequest());
+  }, [dispatch]);
+
+  useEffect(() => {
+    dispatch(fetchGetPostsRequest());
+  }, [dispatch]);
+  return (
+    <>
+      {/*{loading && (*/}
```
Comment (Ruslan1516) [inline src/screen/AllPosts/AllPost.tsx:26]: Why loading is commented?

**Review by Ruslan1516 — APPROVED** [review]

File: src/assets/icon/AddPostIcon.tsx (hunk 1 above)
Comment (RayzigenSchwarz) [reply src/assets/icon/AddPostIcon.tsx]: DONE EPT`

**Pull request merged**